packages/atomic/src/components/search/facets/atomic-facet/e2e/atomic-facet.e2e.ts

```diff
@@ -151,3 +153,158 @@ test.describe('when the "Show more" button has been selected', () => {
     });
   });
 });
+
+const sortCriteriaTests: {
+  criteria: FacetSortCriterion;
+  sortFunction: (values: string[]) => string[];
+}[] = [
+  {
+    criteria: 'alphanumeric',
+    sortFunction: (values: string[]) => [...values].sort(),
+  },
+  {
+    criteria: 'alphanumericDescending',
+    sortFunction: (values: string[]) => [...values].sort().reverse(),
+  },
+  {
+    criteria: 'alphanumericNatural',
+    sortFunction: (values: string[]) =>
+      orderBy([...values], [(value) => value], 'asc'),
+  },
+  {
+    criteria: 'alphanumericNaturalDescending',
+    sortFunction: (values: string[]) =>
+      orderBy([...values], [(value) => value], ['desc']),
+  },
+];
+
+test.describe.serial('Sort Criteria', () => {
```

Comment:
Why serial at the root of the tests ? They don't individually depend on each other. If SAPI had a bug with `alphanumeric` specifically, all 4 would fail when only the first one should fail. Is that how serial works ?